tests/lms/views/test_lti_launches.py

```diff
@@ -33,3 +36,15 @@ def test_render_unauthorized_for_students(self, lti_launch_request, module_item_
         lti_launch_request.params['roles'] = 'urn:lti:role:ims/lis/Learner'
         value = lti_launches(lti_launch_request)
         assert 'This page has not yet been configured' in value.body.decode()
+
+    def test_raises_for_missing_context_id_param(self, lti_launch_request):
+        lti_launch_request.params.pop('context_id', None)
+
+        with pytest.raises(MissingLTILaunchParamError, match="Context Id is required for lti launch."):
```

Comment:
You'll need to update this `match` if you change the error messages as suggested in a previous comment/